cli/src/semdep/parsers/go_sum.py

```diff
@@ -0,0 +1,62 @@
+"""
+Parser for go.sum files
```

Comment:
Please add a link to a spec when possible for each of those lockfile, possible with a quick example.
For example here you can add a link to https://go.dev/ref/mod#go-sum-files

Reply:
I think I failed to find specs for some of these a while ago and gave up, but I'm gonna search again. Thanks for pointing out the Go one. Many of them have a page to the effect of "we have lockfiles" but with no grammar. I'll link those though because it's better than nothing.

Reply:
Ok added everything I could find